File: code_stubs/sort.go
package main

import (
	"fmt"
	"strconv"
	"strings"
)

func printArray(arr []int) {
	fmt.Println("{")
	fmt.Println(strings.Trim(strings.Join(strings.Fields(fmt.Sprint(arr)), " "), "[]"))
	fmt.Println("}")
}

func sortArray(arr []int) {
	// Your code goes here!

	printArray(arr) // Make sure to call this print after every iteration
}

func main() {
	fmt.Print("Enter numbers separated by spaces: ")
	var input string
	fmt.Scanln(&input)

	strs := strings.Split(input, " ")
	var arr []int
	for _, s := range strs {
		num, err := strconv.Atoi(s)
		if err == nil {
			arr = append(arr, num)
		}
	}

	printArray(arr)
	sortArray(arr)
	printArray(arr)
}

File: code_stubs/sort.py
def print_array(arr):
	print("{")
	print(" ".join(map(str, arr)))
	print("}")

def sort(arr):
	# Your code goes here!

	print_array(arr)  # Make sure to call this print after every iteration

if __name__ == "__main__":
	arr = list(map(int, input("Enter numbers separated by spaces: ").split()))

	print_array(arr)
	sort(arr)
	print_array(arr)
	